packages/env/src/env/development.env

```diff
@@ -1,11 +1,11 @@
 NODE_ENV=development
 
-DATABASE_URL=postgres://latitude:secret@db:5432/latitude_development
+DATABASE_URL=postgres://latitude:secret@localhost:5432/latitude_development
 
 ELASTIC_URL=https://elastic:9200
 ELASTIC_USERNAME=latitude
 ELASTIC_PASSWORD=secret
 
 REDIS_PORT=6379
-REDIS_HOST=redis
+REDIS_HOST=localhost
 REDIS_PASSWORD=secret
```

Comment:
Running `web` locally because otherwise my mac takes ~20 seconds to hot-refresh any changes 😭

Reply:
We need to think how to enable host run nextjs for Carlos in a way that can be commited. Maybe a `$USE_WEB_IN_HOST` env variable that’s  sets this to `localhost` or maybe better load the `development.localhost.env` file. Do a template file